components/CardItem.vue

```diff
@@ -147,5 +147,56 @@ export default {
   &.other-card::before {
     background-image: url('@/static/image/cards/other.png');
   }
+
+  &.self-card::after,
```

Comment:
all the styling for .self-card and .other-card relates to classes (and logic) in `SpectrumGameSelfOther.vue`, not in `CardItem` and would thus be better located there than here.

Reply:
That's a good point! thanks